AlphaWallet/Settings/ViewControllers/SaveCustomRpcViewController.swift

```diff
@@ -120,6 +120,7 @@ class SaveCustomRpcViewController: UIViewController {
             }
         }
         editView.allTextFields.first(where: { !$0.statusLabel.text.isEmpty })?.becomeFirstResponder()
+        editView.setNeedsLayout()
```

Comment:
This is needed because when a `TextField` becomes invalid, the layout is changed but `layoutSubViews()` for the `editView` is not called. The `contentSize.width` of the `scrollView` is set to zero again. This forces the `contentSize.width` to be reset to non-zero value.